packit_service/worker/reporting.py

```diff
@@ -100,11 +100,12 @@ def report_status_by_comment(
 
         if isinstance(check_names, str):
             check_names = [check_names]
-        comment_msg = [
+
+        comment_table_rows = ["| | |", "| ------------- | ------------ |"] + [
```

Comment:
The empty header looks a bit weird on GitLab, I cannot seem to find table like that on GitHub in hello-world, what about adding comment like:
```suggestion
        comment_table_rows = ["| Job | Result |", "| ------------- | ------------ |"] + [
```

Reply:
> I cannot seem to find table like that on GitHub in hello-world

Because we don't use comments on GitHub for that. (It's used when we can't set status on MR.)

Reply:
I've added the text header and fix the tests. PTAL